### PR TITLE
Prepare Simplechart readme for wikification

### PR DESCRIPTION
**Pull request by davisshaver**
Refreshes Simplechart readme to link out to Wiki for advanced topics.

Creates a few new pages based on content pulled from readme:

- [Connecting the app to your CMS](https://github.com/alleyinteractive/simplechart/wiki/Connecting-the-app-to-your-CMS)
- [Structuring and populating data for embeds](https://github.com/alleyinteractive/simplechart/wiki/Structuring-and-populating-data-for-embeds)
- [How to add a new chart type](https://github.com/alleyinteractive/simplechart/wiki/How-to-add-a-new-chart-type)

Cleans up the [main Wiki home](https://github.com/alleyinteractive/simplechart/wiki).

Still pending:

- Review/cleanup of [How data flows through the app](https://github.com/alleyinteractive/simplechart/wiki/How-data-flows-throught-the-app)
- Adding React component docs to _How to add a new chart type_

### CODE REVIEW

**Comment by davisshaver** [comment]
Couple specific questions for @joshkadis... 

- Still need to fill out the React section for adding a chart type. Might be the best way to learn this was adding one myself. But not sure which I'd add. Thoughts?

- In the Data Transformer section of https://github.com/alleyinteractive/simplechart/wiki/How-to-add-a-new-chart-type, I wanted to confirm what the object keys are named (or maybe it doesn't matter). Earlier docs called them `foo` & `bar`, but based on some fiddling with Papa Parse, I believe they are getting string keys based on header values.

**Comment by joshkadis** [comment]
Can you move the logo to https://github.com/alleyinteractive/simplechart/tree/master/docs/lib/images

File: README.md [hunk 1]
```diff
@@ -1,161 +1,46 @@
...
+# ![Simplechart](docs/_includes/logo.png)
 
-## Build process
+Quickly create and structure interactive charts for use in a content management system (CMS).
 
-`$ npm run build` will compile both the App and Widget bundles, using the latest Git commit hash as a version.
+*See [Simplechart for WordPress](https://github.com/alleyinteractive/wordpress-simplechart) for the WordPress plugin.*
 
-_Don't use this_ during normal development. You'll only need to build if you're pushing to an implementation of Simplechart, e.g. its [WordPress plugin](https://www.github.com/alleyinteractive/wordpress-simplechart).
+## Quick Start
```
Comment (joshkadis) [inline README.md]: "Quick Start for Local Development"?

File: README.md [hunk 2]
```diff
@@ -1,161 +1,46 @@
...
+Simplechart provides the most utility at present when used in conjunction with the WordPress plugin. You don't need to install this package to try the WordPress plugin.
 
-The widget recognizes Simplechart placeholder divs on a page, fetches data, and renders charts.
+Although we have focused on WordPress to date, Simplechart itself is CMS agnostic, and can even be used without a CMS. Simplechart allows you to control your own data, and customize default styling to maintain visual consistency with your brand.
 
-You can find widget examples at `http://localhost:8080/widget.html`
+Simplechart runs on modern web technology – React, WebPack, and ES5 – and Simplechart widgets are **AMP-compatible** out of the box.
 
-## Adding a new chart type
+Simplechart is and always will be free as in beer, and the code is licensed under [GPL-2.0](https://wordpress.org/about/gpl/).
```
Comment (joshkadis) [inline README.md]: Simplechart is ~~and always will be free as in beer, and the code is~~ licensed under

File: README.md [hunk 3]
```diff
@@ -1,161 +1,46 @@
...
 
-The app is where you input data, select a chart type, set up options. Within an iframe, e.g. when used in the WordPress plugin, the app sends data to the parent window via postMessage for saving.
+Congrats! You should now be able to access Simplechart at `http://localhost:8080/index.html`.
 
-You can see the app at `http://localhost:8080/index.html`
+If you're ready to deploy the app, head over to the Wiki for the [Deployment Guide](TK).
 
-### Widget
+## Use Cases
+Simplechart provides the most utility at present when used in conjunction with the WordPress plugin. You don't need to install this package to try the WordPress plugin.
```
Comment (joshkadis) [inline README.md]: Link to WP plugin repo

**Comment by davisshaver** [comment]
🚀

**Pull request merged**